_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light: photoacoustic radiology improved"
+author: RaoOfPhysics
+---
+
+_Using interstitial light source for tissue-imaging technique based on photoacoustic spectroscopy improves penetration depth and shows promise for minimally invasive radiology_
```

Comment:
Dek is too long and dense. How about "How to see better inside tissue? Shine light from within!"